### PR TITLE
Fix pipeline error in editor.

### PR DESCRIPTION
**Pull request by caryliu1999**
https://github.com/cocos-creator/3d-tasks/issues/4740

### CODE REVIEW

**Comment by pandamicro** [comment]
Please describe how this issue happened

File: cocos/core/renderer/ui/ui-batch-model.ts [hunk 1]
```diff
@@ -69,11 +66,15 @@ export class UIBatchModel extends Model {
...
+        subModel.directInitialize(batch.material!.passes, batch.hInputAssembler, batch.hDescriptorSet!);
     }
 
     public destroy () {
-        this._subModel.destroy();
+        if (this._handle) {
+            ModelPool.free(this._handle);
+            this._handle = NULL_HANDLE;
+        }
         super.destroy();
```
Comment (YunHsiao) [inline cocos/core/renderer/ui/ui-batch-model.ts]: 确定先 destroy handle 再调 super.destroy? 这会有一堆 invalid handle 报错的吧

Comment (caryliu1999) [reply cocos/core/renderer/ui/ui-batch-model.ts]: > 确定先 destroy handle 再调 super.destroy? 这会有一堆 invalid handle 报错的吧

嗯，这里的可以删掉了，已修改

**Comment by caryliu1999** [comment]
> Please describe how this issue happened

Repeated destroy of the subModel caused the problem.

**Review by YunHsiao — APPROVED** [review]

**Pull request merged**